operators/index.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="operators-overview"]
+= Operators overview
+include::modules/common-attributes.adoc[]
+:context: operators-overview
+
+toc::[]
+
+[id="importance-of-operators"]
+
+include::modules/operators-overview.adoc[leveloffset=+1]
+
+With Operators, you an create applications to monitor the running services in the cluster.
+Operators are designed specifically for your applications. Operators implement and automate the common Day 1 operations such as installation and configuration as well as Day 2 operations such as auto-scaling up and down and backups. All these activities are in a piece of software running inside your cluster.
+To understand more about Operators, see xref:../operators/understanding/olm-what-operators-are.adoc#olm-what-operators-are[What are Operators?]
+
+[id="overview-developer-tasks"]
+== For developers
+
+As a developer, you can perform the following Operator tasks:
+
+** xref:../operators/operator_sdk/osdk-installing-cli.adoc#osdk-installing-cli[Install Operator SDK CLI].
+** xref:../operators/user/olm-creating-apps-from-installed-operators.adoc#olm-creating-apps-from-installed-operators[Create an application from an installed Operator through the web console].
+** xref:../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Install and subscribe an Operator to your namespace].
+** xref:../operators/operator_sdk/osdk-about.adoc#osdk-about[Use Operator SDK to build,test, and deploy an Operator].
+** Create xref:../operators/operator_sdk/golang/osdk-golang-quickstart.adoc#osdk-golang-quickstart[Go-based Operators], xref:../operators/operator_sdk/ansible/osdk-ansible-quickstart.adoc#osdk-ansible-quickstart[Ansible-based Operators], and xref:../operators/operator_sdk/helm/osdk-helm-quickstart.adoc#osdk-helm-quickstart[Helm-based Operators].
+
+[id="overview-administrator-tasks"]
+== For administrators
+
+As a cluster administrator, you can perform the following Operator tasks:
```

Comment:
The items with correct order:

- [Manage custom catalogs](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/admin/olm-managing-custom-catalogs.html#olm-managing-custom-catalogs)
- [Allow non-cluster administrators to install Operators](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/admin/olm-creating-policy.html#olm-creating-policy)
- [Install an Operator from OperatorHub](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/user/olm-installing-operators-in-namespace.html#olm-installing-operators-in-namespace)
- [View Operator status](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/admin/olm-status.html#olm-status).
- [Manage Operator conditions](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/admin/olm-managing-operatorconditions.html#olm-managing-operatorconditions)
- [Upgrade installed Operators](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/admin/olm-upgrading-operators.html#olm-upgrading-operators)
- [Delete installed Operators](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/admin/olm-deleting-operators-from-cluster.html#olm-deleting-operators-from-a-cluster)
- [Configure proxy support](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/admin/olm-configuring-proxy-support.html#olm-configuring-proxy-support)
- [Use Operator Lifecycle Manager on restricted networks](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/admin/olm-restricted-networks.html#olm-restricted-networks)